lms/services/youtube.py

```diff
@@ -90,7 +90,9 @@ def factory(_context, request) -> YouTubeService:
     app_settings = request.registry.settings
 
     return YouTubeService(
-        enabled=ai_settings.get("youtube", "enabled", True),
+        enabled=ai_settings.get_setting(
+            ai_settings.fields[ai_settings.Settings.YOUTUBE_ENABLED]
```

Comment:
If `Settings` was a top-level class in `ApplicationSettings` it could just be imported directly, which would make expressions like this a bit more succinct (`ai_settings.fields[Settings.YOUTUBE_ENABLED]`).